resources/tiers.md

```diff
@@ -0,0 +1,84 @@
+# SDK tiers
+
+The objective of Meilisearch's Integrations Team is to **ease Meilisearch usage to as many developers as possible**.
+
+Given this statement, **tiers** are nothing more than a __group of SDKs/integrations__ where the integrations team can fairly divide their attention. Unfortunately is not possible to keep all the SDKs 100% updated against the new Meilisearch releases all the time, the Integrations Team is a polyglot small team that maintains 30+ different integrations and tools, so it is humanly impossible to consistently meet the Meilisearch quality standards.
+
+The ultimate goal of introducing tiers is to allow the team to have more time for new projects/integrations so that more time can be invested and more users will be impacted.
+
+It's been using [data collected here](https://docs.meilisearch.com/learn/what_is_meilisearch/telemetry.html) and data like GitHub stars, forks, and watchers to measure the impact of an SDK.
+
+It is worth remembering that this priority is just a concept introduced by the team where it aims to give a decision weight before
+we invest time in some issues of some SDK in question.
+
+:warning: We will keep all officially maintained SDKs **working**. :warning:
+
+## Tier #1
+
+The SDKs in group #1 have total priority over the team's time. Issues are visited before issues are present in tier #2, for example.
+It is worth remembering that this does not mean open issues will be finalized promptly. Every issue needs time to be understood/confirmed/worked on, and consequently closed.
+All new Meilisearch features that make sense for those SDKs will be available in the release day.
+
+## Tier #2
+
+Tier #2 SDKs are the emergent ones, we strongly believe they can move further, and we are betting on them.
+In any case, since they do not bring that much value compared to tier #1 we will only apply the amount of time required to keep it working and do important maintenance code and code cleanup.
+
+Only in urgent cases where, for example, the SDK is no longer working issues from the SDKs in group #2 will be prioritized in comparison to tier #1.
+
+
+## Tier #3
+
+The SDKs in group #3 have relatively low user adoption compared to groups #2 and #1. They are probably new or have a small community behind the tech stack in the first place.
+As with tier #2 SDKs in this group will be prioritized only in critical cases, and they are not going to receive active improvements from the integrations team.
```

Comment:
```suggestion
As with tier #2, SDKs in this group are prioritized only in critical cases, and they are not going to receive active improvements or maintance from the integrations team.
```

Active improvements?

Reply:
You know, after the introduction of the tiers, I'll not spend time going through the Swift SDK looking for ways of improving the code, since I can do it in Ruby (which is tier `#2`), or even in the PHP `#1`.